src/js/components/RangeSelector/RangeSelector.js

```diff
@@ -183,6 +183,9 @@ if (process.env.NODE_ENV !== 'production') {
   doc(RangeSelector);
 }
 
-export default compose(
+const WrappedRangeSelector = compose(
   withTheme,
 )(RangeSelector);
+
+export default React.forwardRef((props, ref) =>
```

Comment:
how about we do a HOC called `withForwardRef` and we do this once inside that hoc? with that we can keep using compose + export default.